setup/lib/utils.js

```diff
@@ -0,0 +1,32 @@
+const { setFailed } = require('@actions/core');
+const os = require('os');
+const path = require('path');
+const execSync = require('child_process').execSync;
+const semver = require('semver');
```

Comment:
Looks like these two are not used. We can remove them.

Reply:
Removed with the latest commit, thanks for catching that. :)